inventory/upcloud.ini

```diff
@@ -20,3 +19,4 @@
 return_ip_addresses = True
 return_non_fqdn_names = False
 default_timeout = 300
+default_ipv_version = IPv4
```

Comment:
Needed to fix issue https://github.com/UpCloudLtd/upcloud-ansible/issues/3 (specify preffered internet protocol version to use )